package.json

```diff
@@ -6,7 +6,7 @@
 		"@axa-fr/react-oidc": "^7.22.32",
 		"@fontsource/source-sans-pro": "^5.1.0",
 		"@heroicons/react": "^2.1.3",
-		"@pdc/sdk": "^0.11.1",
+		"@pdc/sdk": "^0.16.1",
```

Comment:
Right now, this upgrade happens in the first commit, and therefore the first commit does not build.

Instead, let's gradually bump the SDK version along with the changes necessary for that specific version of the SDK. I believe, in order, they are: handle timestamp strings, include a source id with bulk uploads, and rename organizations to changemakers.

Reply:
@jasonaowen Egads! I've hit an unfortunate issue here -- the SourceBundle, which is needed for the source handling, was missed in the openapi spec and I had to add it back in (https://github.com/PhilanthropyDataCommons/service/pull/1281) at a later date. As such, an older version of the pdc cannot be used for the sourceId commit without also refactoring many organizations to changemakers. Thoughts on either merging the two commits? That seems messy but then again, the changes are separate enough that they might still be discnerable

Reply:
My .02 which may not be agreed on here is that it is appropriate to have one commit that bumps to the latest version along with all required changes to make that bump.

I know this is not ideal, but also I think this is the price of allowing versions to bump so much!

Alternatively Harry you could decide to take Jason's approach and bump one version at a time.  I'm just not sure that's worth the extra dev time really.